src/applications/simple-forms/shared/components/confirmationPageViewHelpers.jsx

```diff
@@ -248,4 +293,80 @@ export const ChapterSectionCollection = ({
       )
     );
   });
+
+  if (collapsible && content.length) {
+    return (
+      <div className={className || 'vads-u-margin-top--2'}>
+        <div className="print-only">{content}</div>
+        <div className="screen-only">
+          <VaAccordion bordered open-single uswds>
+            <VaAccordionItem header={header} id="info" bordered uswds>
+              {content}
+            </VaAccordionItem>
+          </VaAccordion>
+        </div>
+      </div>
+    );
+  }
+
+  return content;
+};
+
+export const useDevOnlyButtons = ({
```

Comment:
decided to add this to help with easy testing on confirmation page. it's optional.